packages/katello/katello/service-wait

```diff
@@ -57,25 +53,13 @@ before_start() {
 # after start or restart
 after_start() {
   case "$SERVICE" in
-    tomcat|tomcat6)
-      # RHBZ 789288 - wait until data port is available
-      wait_for_url $TOMCAT_TEST_URL
-      ;;
     foreman)
       wait_for_url $FOREMAN_TEST_URL
       ;;
     katello)
```

Comment:
I wouldn't mind getting rid of this. In production we don't have katello as a standalone service. This probably dates back to the time katello was a standalone rails application rather than a foreman plugin.